lua/neogit/lib/git/status.lua

```diff
@@ -195,17 +195,17 @@ end
 
 local git = { cli = require("neogit.lib.git.cli") }
 local status = {
-  stage = function(...)
-    git.cli.add.files(...).call()
+  stage = function(files)
+    git.cli.add.files(unpack(files)).call()
```

Comment:
I like how the `...` looks more, but thats not a suggestion to change it back. Is there an advantage to one style over the other, or is it just a preference?

Reply:
Not really, we need an unpack somewhere since we collect the files to stage in a list.

I really want to add this to just a list of arguments, but I do believe vim.system may be stable now, so I'll tackle that next if the case